Directory.Build.props

```diff
@@ -48,7 +48,7 @@
     <DotNetCmd></DotNetCmd>
   </PropertyGroup>
 
-  <Target Name="prepack" DependsOnTargets="Build" Condition="'$(IsPackable)' == 'true'">
+  <Target Name="prepack" DependsOnTargets="Build" Condition="'$(IsPackable)' == 'true' AND '$(RID)' != 'win-x86'">
```

Comment:
Perhaps its better to just use `$(PlatformTarget)' == 'x64'` instead. We can discuss how to deploy more configurations when the the time comes.